docs/agent/changes.md

```diff
@@ -374,13 +377,110 @@ in almost anyone's checks. The flare module, for example, was removed and
 reimplemented in Go, but is unlikely to have been used by anyone in a custom check.
 To learn more, you can read about the details in the [development documentation][python-dev].
 
+### Agent Integrations 
+
+Even if the new Agent fully supports Python checks, a number of those provided
+by [integrations-core](https://github.com/DataDog/integrations-core) are expected to fail if run within the
+Agent as they are not currently implemented:
+
+* agent_metrics
+* docker_daemon [replaced by a new `docker` check](#docker-check)
+* kubernetes [replaced by new checks](#kubernetes-support)
+
+### Check API
+
+Some methods in the `AgentCheck` class are not currently implemented. These include:
+
+* `service_metadata`
+* `get_service_metadata`
+* `generate_historate_func`
+* `generate_histogram_func`
+* `stop`
+
+### Custom Checks
+
+If you happen to use custom checks, there's a chance your code depends on py code
+that was bundled with agent5 that may not longer be available in the with the new
+agent 6 package. This is a list of packages no longer bundled with the agent:
+
+- backports.ssl-match-hostname
+- boto
+- certifi
+- chardet
+- datadog
+- decorator
+- future
+- futures
+- google-apputils
+- pycurl
+- pyOpenSSL
+- python-consul
+- python-dateutil
+- python-etcd
+- python-gflags
+- pytz
+- pyvmomi
```

Comment:
pretty sure this is bundled through the vsphere check

Reply:
Good catch, removed. I did another pass of the check and removed anything that we currently include.